src/stores/RoomViewStore.js

```diff
@@ -23,16 +23,18 @@ import { _t } from '../languageHandler';
 const INITIAL_STATE = {
     // Whether we're joining the currently viewed room
     joining: false,
-    // Any error occurred during joining
+    // Any error that has occurred during joining
     joinError: null,
-    // The room ID of the room
+    // The room ID of the room currently being viewed
     roomId: null,
-    // The event being viewed
-    eventId: null,
-    // The offset to display the event at (see scrollStateMap)
-    eventPixelOffset: null,
-    // Whether to highlight the event
-    isEventHighlighted: false,
+
+    // The event to scroll to initially
```

Comment:
could you clarify 'initially' here: I think you mean 'when the room is first viewed' or sth?